src/Lumi/Components/Form.purs

```diff
@@ -352,6 +355,25 @@ passwordBox
        String
 passwordBox = inputBox Input.password
 
+-- | A configurable textarea box makes a `FormBuilder` for strings
```

Comment:
In the HTML docs, this won't necessarily have the line break in the same place, in which case it might read a bit strangely. You could add a full stop after "strings" perhaps?

Reply:
The style `white-space: pre-wrap;` will fix this. `pre` keeps the newlines and `wrap` adds back the normal text wrapping behavior for paragraphs.